package.json

```diff
@@ -190,8 +191,12 @@
     "nodemon": "^2.0.7",
     "npm-check": "^6.0.1",
     "npm-run-all": "^4.1.5",
+    "patch-package": "^8.0.0",
+    "pixelmatch": "^5.3.0",
+    "pngjs": "^7.0.0",
```

Comment:
TODO: Get @lesterchoi-okta thoughts on this. Lee suggested that we could maybe disable the test suites but keep the code in case it is useful in the future?